src/SampleDropdown.tsx

```diff
@@ -11,37 +10,53 @@ function SampleDropdown() {
   const loadSample = useAppStore((state) => state.loadSample);
 
   const items = samples.map((s) => ({
-    label: s.NAME,
+    text: s.NAME,
+    value: s.NAME,
     key: s.NAME,
   }));
 
-  const handleMenuClick = async (e: any) => {
-    if (e.key) {
+  const handleDropdownChange = async (
+    _event: React.SyntheticEvent<HTMLElement>,
+    data: DropdownProps
+  ) => {
+    const { value } = data;
+    if (typeof value === "string") {
       setLoading(true);
       try {
-        await loadSample(e.key);
-        message.info(`Loaded ${e.key} sample`);
+        await loadSample(value);
```

Comment:
Does this mean we have to close your old PR where you had fixed this issue?

Reply:
yeah, basically this solves that too. that PR can be closed